frontend/src/widgets/RttapaUpdates.js

```diff
@@ -42,7 +42,11 @@ export default function RttapaUpdates({
             <tbody>
               {logs.map((log) => (
                 <tr key={`logrow${log.id}`}>
-                  <td data-label="Date">{log.data ? log.data.communicationDate : ''}</td>
+                  <td data-label="Date">
+                    <Link to={`/recipient-tta-records/${recipientId}/region/${regionId}/communication/${log.id}/view`}>
+                      {log.data ? log.data.communicationDate : ''}
```

Comment:
I know this is the way it was before, do we need to worry about the log.data.communicationDate being blank and not having a link here? Do we want a default value just in case.

Reply:
I've already discussed this with UX. There is currently no way to save a log entry without a date, so no need for a fallback here. If it turns out to be an issue we can circle back.